packages/packages/commonwell-cert-runner/src/document-consumption.ts

```diff
@@ -23,7 +23,7 @@ export async function queryDocuments(
 
   const { personId, patientId } = await findOrCreatePerson(commonWell, queryMeta, makeDocPerson());
 
-  if (!personId) throw new Error(`[E1c] personId is undefined before calling getNetworkLinks()`);
+  if (!personId) throw new Error(`[E1c] personId is undefined before calling getPatientLinks()`);
```

Comment:
renamed but I think the previous one was the correct name?

Reply:
Must have been a bad merge, i updated it